modules/nw-ovn-k-anp-best-practices.adoc

```diff
@@ -0,0 +1,58 @@
+/ Module included in the following assemblies:
```

Comment:
nit: this is showing up in the preview, not sure if it was intended to

Reply:
thank you.